templates/named.conf.options.erb

```diff
@@ -62,6 +62,12 @@ check-names slave <%= @check_names_slave %>;
 check-names response <%= @check_names_response %>;
 <% end -%>
 
+<% if @statistic_channel_ip and @statistic_channel_ip -%>
```

Comment:
ok, wanna have test for the other validation too ? I could add them in another pull request.